metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -247,7 +274,10 @@ def can_parse_entry(cls, entry: AuditLogEntry) -> bool:
 
     @classmethod
     def from_entry(cls, entry: AuditLogEntry) -> "QueryEvent":
-        user = entry.payload["authenticationInfo"]["principalEmail"]
+        try:
+            user = entry.payload["authenticationInfo"]["principalEmail"]
+        except (KeyError, TypeError):
+            user = "unknown"
```

Comment:
Need to change this piece :)  -- we don't want to report unknowns

Reply:
Cool, removed the try catch since we have one below when parsing QueryEvents anyways!